lib/src/utils/apple_pay_utils.dart

```diff
@@ -0,0 +1,12 @@
+import 'dart:convert';
+
+import 'package:flutter/services.dart';
+
+const paymentConfigFile = "assets/default_payment_profile_apple_pay.json";
```

Comment:
1. should be inside `ApplePayUtils`
2. should be private

Reply:
Unresolved + could you add type
`static const String paymentConfigFile = "assets/default_payment_profile_apple_pay.json";`

Reply:
Still not fixed

Reply:
Still not fixed